app/forms/living_situation_form.rb

```diff
@@ -0,0 +1,12 @@
+class LivingSituationForm < Step
+  def self.application_attributes
+    [:living_situation]
+  end
+
+  form_attributes(*application_attributes)
```

Comment:
How would you feel about either not defining application_attributes (since we don't use in the controller) or using something like application_params (rather than form_params) to update the model in the controller?

Reply:
I like using application_params in the controller. Just added that in 279c4aa3b5d02ae4be59a28b30afc017480da779.

It's more clear: "I'm updating the application, so I should use the application_params."